### DISCUSSION TITLE
%E5%B0%8E%E8%AE%80%E5%8D%A1/%E7%AC%AC28%E9%80%B1%E5%B0%8E%E8%AE%80%E5%8D%A1

### DISCUSSION THREAD

**Discussion by giscus** [cit-hall-43/cit-hall-43.github.io#2] [Q&A]
# %E5%B0%8E%E8%AE%80%E5%8D%A1/%E7%AC%AC28%E9%80%B1%E5%B0%8E%E8%AE%80%E5%8D%A1

晨興補充材料 出埃及記生命讀經 第五至六篇–神呼召所豫備的人 二O二四年七月八日至十三日

https://cit-hall-43.github.io/%E5%B0%8E%E8%AE%80%E5%8D%A1/%E7%AC%AC28%E9%80%B1%E5%B0%8E%E8%AE%80%E5%8D%A1.html

**Comment by cit-hall-43** [comment]
凡事倚靠，凡事謝恩
